src/Controller/PageController.php

```diff
@@ -129,7 +142,7 @@ public function page(string $slug, Request $request)
         // request to option cards api
         $optionCardsUrl = getenv('APP_API_BASE_URL') . 'ccs/v1/option-cards/0';
 
-        $response = $client->request(
+        $response = $this->client->request(
```

Comment:
Nice. I think I may have to refactor code for homepage aswell to have it like this as it looks a little bit cleaner